### ISSUE TITLE
Incomplete instruction on how to add content script

### ISSUE THREAD

**Issue by mkarang** [mingyaulee/Blazor.BrowserExtension#30]
Thank you very much for your work. Everything works until I tried following the instruction to add content script. I get the following error:

The type or namespace name 'IBrowserExtensionEnvironment' could not be found (are you missing a using directive or an assembly reference?)

The error above comes from App.razor, on the line:  "@inject IBrowserExtensionEnvironment BrowserExtensionEnvironment"

**Comment by mkarang** [comment]
Thank you very very much for your great work and for sharing. I started using your Sidebar project before I saw you fixed this. 

That Sidebar Blazor example is super useful. With that, I also managed to figure out how to call csharp from js (within app.js).

**Comment by mingyaulee** [comment]
Glad that it helped. If you have any other idea of what can be included in the samples it would be great.

**Comment by mkarang** [comment]
Thank you, you're super great! Yes I do have one idea: make Sidebar work with NET 6. With NET 6 we can have AOT which will make wassm runs much faster than javascript, React, etc. This will enable running computationally expensive things in Chrome extension using C#.

I tried converting your Sidebar project to NET6 with AOT. After getting successful build, I was able to run it in chrome but only app.js is working. The blue Toggle button doesn't show up and I got this error on chrome developer console: 

dotnet.6.0.0.tj42mwroj7.js:1 Uncaught (in promise) ReferenceError: bodyJs is not defined
    at Object.bind_method (dotnet.6.0.0.tj42mwroj7.js:1)
    at bind_runtime_method (dotnet.6.0.0.tj42mwroj7.js:1)
    at Object.bindings_lazy_init (dotnet.6.0.0.tj42mwroj7.js:1)
    at Object.bind_static_method (dotnet.6.0.0.tj42mwroj7.js:1)
    at Qe (blazor.webassembly.js:1)
    at blazor.webassembly.js:1
    at blazor.webassembly.js:1
    at callRuntimeCallbacks (dotnet.6.0.0.tj42mwroj7.js:1)
    at postRun (dotnet.6.0.0.tj42mwroj7.js:1)
    at doRun (dotnet.6.0.0.tj42mwroj7.js:1)

--------------------------
On chrome extension menu where I load unpacked extension, I see this error:

Uncaught (in promise) ReferenceError: bodyJs is not defined
Context
https://stackoverflow.com/questions/48523118/wasm-module-compile-error-in-chrome-extension <-- this just happens to be a page I was looking at.
Stack Trace
framework/dotnet.6.0.0.tj42mwroj7.js:1 (bind_method)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (bind_runtime_method)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (bindings_lazy_init)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (bind_static_method)
framework/blazor.webassembly.js:1 (Qe)
framework/blazor.webassembly.js:1 (anonymous function)
framework/blazor.webassembly.js:1 (anonymous function)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (callRuntimeCallbacks)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (postRun)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (doRun)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (run)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (runCaller)
framework/dotnet.6.0.0.tj42mwroj7.js:1 (removeRunDependency)
framework/blazor.webassembly.js:1 (h)
